Sources/SWBWindowsPlatform/Plugin.swift

```diff
@@ -25,22 +26,42 @@ struct WindowsPlatformSpecsExtension: SpecificationsExtension {
     }
 }
 
+private func findLatestInstallDirectory(fs: any FSProxy) async throws -> Path? {
+    if try ProcessInfo.processInfo.hostOperatingSystem() == .windows {
+        let installations = try await VSInstallation.findInstallations(fs: fs)
+            .sorted(by: { $0.installationVersion > $1.installationVersion })
+        if let latest = installations.first {
+            let msvcDir = latest.installationPath.join("VC").join("Tools").join("MSVC")
+            let versions = try fs.listdir(msvcDir).map { try Version($0) }.sorted { $0 > $1 }
```

Comment:
Based on recent lessons learned... we should not use the latest version. Instead, locate this file:

```swift
latest.installationPath.join("VC").join("Auxiliary").join("Build").join("Microsoft.VCToolsVersion.default.txt")
```

...and then read its contents (stripping whitespace) to determine the value of `latestVersion`.

This is what vswhere and clang will do, so we'll want to match. _Usually_ this is the same as the latest version, but sometimes it's not, and the "latest" code will fail catastrophically if that's the case. :)

You should also check the `VCToolsVersion` environment variable and use that in preference to `Microsoft.VCToolsVersion.default.txt` if it exists.

Reply:
I also realize this existing code (that I wrote) that you're just moving unchanged, so if you'd rather I fix this in a separate PR I'm also happy to do that, especially so it won't invalidate your CI run. Let me know your preference.

Reply:
Hmm, or we could actually run vcvarsall which is probably better than trying to construct paths manually. Actually, let me just take a look at this separately.